test/e2e/provisioning/helper.go

```diff
@@ -50,7 +50,7 @@ var (
 	}
 
 	openStackImages = map[string]string{
-		string(providerconfigtypes.OperatingSystemUbuntu):     "kubermatic-ubuntu",
+		string(providerconfigtypes.OperatingSystemUbuntu):     "ubuntu-24.04",
```

Comment:
Please add a prefix for this image, preferably `kubermatic-ubuntu` or `kubermatic-ubuntu-24.04`